src/components/event/EventFilter/Event-filter.js

```diff
@@ -52,157 +63,165 @@ class EventFilter extends Component {
     }
   }
 
+  handlerViewMore = () => this.setState(state => !state.viewMore);
+
   render() {
-    const { allCategories, formValues, currentUser } = this.props;
+    const {
+      allCategories,
+      formValues,
+      currentUser,
+      handleSubmit,
+      onReset,
+      initialize,
+      submitting,
+      onLoadUserDefaults,
+    } = this.props;
+    const { viewMore } = this.state;
+
     const values = formValues || { selectedPos: {} };
     const options = [
-      { value: eventStatusEnum.Active, text: "Active" },
-      { value: eventStatusEnum.Blocked, text: "Blocked" },
-      { value: eventStatusEnum.Canceled, text: "Canceled" },
+      { value: eventStatusEnum.Active, text: OPTIONS_ACTIVE },
+      { value: eventStatusEnum.Blocked, text: OPTIONS_BLOCKED },
+      { value: eventStatusEnum.Canceled, text: OPTIONS_CANCELED },
     ];
 
     return (
-      <>
-        <div className="sidebar-filter">
-          <form onSubmit={this.props.handleSubmit} className="box">
-            <div className="form-group">
-              <Field
-                name="keyWord"
-                component={renderTextField}
-                type="input"
-                label="Keyword"
-              />
-            </div>
-            {this.state.viewMore && (
-              <>
-                <div className="form-group">
-                  <Field
-                    name="dateFrom"
-                    label="From"
-                    minValue={new Date()}
-                    component={renderDatePicker}
-                    parse={parseEuDate}
-                  />
-                </div>
-                <div className="form-group">
-                  <Field
-                    name="dateTo"
-                    label="To"
-                    minValue={new Date(values.dateFrom)}
-                    component={renderDatePicker}
-                    parse={parseEuDate}
-                  />
-                </div>
-                <div className="form-group">
+      <div className="sidebar-filter">
+        <form onSubmit={handleSubmit} className="box">
+          <div className="form-group">
+            <Field
+              name="keyWord"
+              component={renderTextField}
+              type="input"
+              label="Keyword"
+            />
+          </div>
+          {viewMore && (
+            <>
```

Comment:
Please check if we can remove fragment tag

Reply:
No, we can`t.